ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -96,6 +96,28 @@ def reshape(self, *args, shape=None):
     def dim(self):
         return self.ivy_array.ndim
 
+    @with_supported_dtypes(
```

Comment:
The issue may be related to this, and you may want to change this to `unsupported` and check if tests pass on the CI